src/main/java/net/atos/zac/util/indexRewriteFilter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.util;
+
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+
+@WebFilter(filterName = "indexRewriteFilter")
+public class indexRewriteFilter implements Filter {
+
+    private final List<String> resourcePaths =
+            Arrays.asList("/assets", "/rest", "/websocket");
```

Comment:
Is het niet beter om hier ook een static van te maken.
En meer gebruikelijk is List.of("A", "B"); i.p.v. Arrays.asList()
En aangezien je later een Stream nodig hebt kun je wellicht ook doen 
static final Stream<String> RESOURCE_PATHS = Stream.of("A", "B");
Ik weet niet of dat werkt

Reply:
Een stream kan je maar 1 keer gebruiken, dus heb er list.of van gemaakt